javascript/node/index.js

```diff
@@ -0,0 +1,111 @@
+import {
+    createRequest,
+    isEmptyMessage,
+    streamBlocks,
+    createAuthInterceptor,
+    createRegistry,
+    fetchSubstream
+} from '@substreams/core';
+import { createConnectTransport } from "@connectrpc/connect-node";
+import fs from "fs";
+
+const TOKEN = process.env.SUBSTREAMS_API_TOKEN
+const ENDPOINT = "https://mainnet.eth.streamingfast.io"
+const SPKG = "https://storage.googleapis.com/substreams-registry/spkg/ethereum-explorer-v0.1.1.spkg"
+const MODULE = "map_block_meta"
+const CURSOR_FILE = "cursor"
+const START_BLOCK = '100000'
+const STOP_BLOCK = '+10000'
+
```

Comment:
I think we should put the `main` function here so that you get the juicy details first, the helper functions fetchPackage and cursor related ones would go under `main`.